docs/openstack/edpm_adoption.md

```diff
@@ -302,6 +267,202 @@ done
     ```
 
 * Wait for the dataplane node set to reach the Ready status:
+
     ```
     oc wait --for condition=Ready osdpns/openstack --timeout=30m
     ```
+
+## Nova compute services fast-forward upgrade from Wallaby to Antelope
+
+Nova services rolling upgrade cannot be done during adoption,
+there is in a lock-step with Nova control plane services, because those
+are managed independently by EDPM ansible, and Kubernetes operators.
+Nova service operator and OpenStack Dataplane operator ensure upgrading
+is done independently of each other, by configuring
+`[upgrade_levels]compute=auto` for Nova services. Nova control plane
+services apply the change right after CR is patched. Nova compute EDPM
+services will catch up the same config change with ansible deployment
+later on.
+
+> **NOTE**: Additional orchestration happening around the FFU workarounds
+> configuration for Nova compute EDPM service is a subject of future changes.
+
+* Configure pre-FFU workarounds for Nova compute EDPM services to update its version records:
+
+    ```yaml
+    oc apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: nova-compute-workarounds
+      namespace: openstack
+    data:
+      19-nova-compute-cell1-workarounds.conf: |
+        [workarounds]
+        disable_compute_service_check_for_ffu=true
+    EOF
+
+    oc apply -f - <<EOF
+    apiVersion: dataplane.openstack.org/v1beta1
+    kind: OpenStackDataPlaneService
+    metadata:
+      name: nova-compute-workarounds
+      namespace: openstack
+    spec:
+      label: nova.compute.workarounds
+      configMaps:
+        - nova-compute-workarounds
+      playbook: osp.edpm.nova
```

Comment:
This is an incomplete service definition. The nova service needs the original two secrets provided as well
```
spec:
  label: dataplane-deployment-nova
  playbook: osp.edpm.nova
  secrets:
  - nova-cell1-compute-config
  - nova-migration-ssh-key
```

Reply:
I would say, those must be only added for the main osdpd/openstack instead. Those extra osdpd have nothing to do to those secrets

Reply:
So, I'm going to put this into https://github.com/openstack-k8s-operators/data-plane-adoption/blob/3a00eeb3e03204b29c3dd52ad25e186ff66d184e/docs/openstack/edpm_adoption.md#L313
```
  secrets:
  - nova-cell1-compute-config
  - nova-migration-ssh-key
```
I would not put it for nova-compute-ffu and nova-compute-workarounds as deploying those have nothing to those secrets. Did I get it right?

Please could you also help to document anythings about  creating nova-cell1-compute-config ? It just pops out of nothere here

Reply:
I see the problem now: "(EDPM) standard, shipped, and packaged DataPlaneServices can not be modified".
So I will add a nova_extra_config service then, and to put these secrets there.

Reply:
I mean each DataPlaneService that runs the nova playbook should have all the configuration input. In my mind DataPlaneService deployemants are not additons to each other but replacement of each other.  The ansible code should (and maybe doesn't today) ensure that the configuration in /etc/nova/nova.conf.d/ is the same as the configuration files provided from the secrets and CMs in the DataPlaneService *being deployed*. Otherwise i) we don't have a way to remove config files ii) we don't use ansible in an idempotent way

Reply:
done also for FFU https://github.com/openstack-k8s-operators/data-plane-adoption/compare/314ca8d40a836ee15daa93e129f3add1080b299c..e86a0510694af44742b1d93f958e93b544c13cce